app/database_etl/etl_main.py

```diff
@@ -89,12 +89,15 @@ def main():
         research_source, research_source_cols = create_research_source_df(dashboard_source)
 
         # Compute pin information for each record in dashboard source table
+        # modify dashboard source structure so country col is similar to city and state col
+        modified_dashboard_source = dashboard_source
+        modified_dashboard_source['country'] = modified_dashboard_source['country'].apply(lambda x: [x])
```

Comment:
Is this best done here, or with a type check later on to see whether the item in question is a list or str?

Reply:
The purpose of this is to get the country field to be similar in structure with the city and state fields. I did this to simplify the logic in the function used to get lats and longs.

Originally the country col would always be a string, just making it a one element list for consistency

Reply:
this makes sense to me